autoload/db_ui/schemas.vim

```diff
@@ -23,11 +23,18 @@ let s:postgres_foreign_key_query = "
       \       ON ccu.constraint_name = tc.constraint_name
       \ WHERE constraint_type = 'FOREIGN KEY' and kcu.column_name = '{col_name}' LIMIT 1"
 
+let s:postgres_list_schema_query = "
```

Comment:
I tried this query as it is, on Postgres 9.5, and I get this error:
```
ERROR:  column "schema_name" does not exist
LINE 6:     schema_name !~ '^pg_'
```
Changing all `schema_name` to `nspname` after `FROM` clause makes everything work. I'm not sure how select aliases work for you in the "where" part, it doesn't work for me.

Reply:
Yes, sorry I tested it in Redshift which has a more "permissive" syntax than pure Postgres, I've changed it to `nspname` and worked for me too, I'll push the changes.